jenkins-pipelines/longevity-alternator-3h.jenkinsfile

```diff
@@ -12,5 +12,5 @@ longevityPipeline(
     test_config: 'test-cases/longevity/longevity-alternator-3h.yaml',
 
     timeout: [time: 300, unit: 'MINUTES'],
-    email_recipients: '["qa@scylladb.com", "alternator@scylladb.com"]'
+    email_recipients: '["qa@scylladb.com","alternator@scylladb.com"]'
```

Comment:
ditto, unrelated to dns server :-)
But I don't know what your patch process is. Maybe you consider it fine to mix unrelated patches into one series like this.

Reply:
not related directly, but was annoying me while testing this.